### ISSUE TITLE
ucrtbased.dll is missing from your computer.

### ISSUE THREAD

**Issue by Unknown025** [hfiref0x/UACME#15]
Compiled the source code on Windows 7, copied the executable to Windows 10 and the executable refuses to launch ("The program can't start because ucrtbased.dll is missing from your computer. Try reinstalling the program to fix this problem.").

Also small side note, are there any current UAC exploits that do not require an administrator token to function?

Thanks.

**Comment by hfiref0x** [comment]
You compiled in Debug configuration. There is no "ucrtbased.dll" in Release.

**Comment by Unknown025** [comment]
Hm...when I tried to compile in "Release" configuration, Visual Studio 2017 refused to compile. Also, in "Debug" configuration, on Windows 7, the executable worked just fine. I can't really tell why it's not working on Windows 10. When I added the ucrtbased.dll to the working directory (I looked up the error, and apparently it's an issue with Visual Studio not copying dlls to the right file paths), the program crashed with error 0x000007b, although from the looks of things, it seems like the error is system specific and not related to the source code.
